test/strings/strcat.c

```diff
@@ -0,0 +1,17 @@
+#include <string.h>
+#include "smack.h"
+
+// @expect verified
+
+int main(void) {
+  char brick[10];
+  strcpy(brick,"bl");
+  
+  char *glue = "ue";
+
```

Comment:
No need for this new line. You can make these codes tighter in terms of new lines.